### ISSUE TITLE
webpacks's `ignoreWarnings` is not used

### ISSUE THREAD

**Issue by ImLunaHey** [RoccoC/webpack-build-notifier#90]
I noticed I'm still getting notifications for warnings that're being filtered by webpack's `ignoreWarnings` field. 🤔

**Comment by nolimitdev** [comment]
The same problem here :( Also [webpack-build-notifier](https://github.com/RoccoC/webpack-build-notifier) does not support messages filtering to omit it.

**Comment by RoccoC** [comment]
There's an [open PR](https://github.com/RoccoC/webpack-build-notifier/pull/91) to fix this, but IIRC tests are failing. Happy to merge it in once the issues there are fixed! 🙏 cc: @ImLunaHey

**Comment by ImLunaHey** [comment]
this has been working in production for a while now. which tests aren't passing?

**Comment by RoccoC** [comment]
Fixed in #91 and #92.

**Comment by Drumstix42** [comment]
In my configuration, to use v3.0.0, I needed update my import from this:

```
const WebpackBuildNotifierPlugin = require('webpack-build-notifier');
```

to:
```
const WebpackBuildNotifierPlugin = require('webpack-build-notifier').default
```

Is that expected?

**Comment by RoccoC** [comment]
> Is that expected?

No, it isn't. Looks like this is because of how `tsup` bundles commonjs. Let me get this fixed...

**Comment by RoccoC** [comment]
@Drumstix42 , this is fixed in [3.1.0](https://github.com/RoccoC/webpack-build-notifier/releases/tag/3.1.0). Thanks for bringing it to my attention!

**Comment by nolimitdev** [comment]
Confirming that issue is fixed. Thanks all! Great job!
